### PR TITLE
Add support for Uber for Business APIs

### PR DESCRIPTION
**Pull request by dustinwhittle**

### CODE REVIEW

**Comment by christinekim** [comment]
Synced with Dustin: tests are difficult since there is no way to pull real trips from these endpoints with our U4B accounts. Working with U4B team to get a test environment so we can release w/ tests

File: tests/test_auth.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def client_credential_oauth2credential():
         access_token=ACCESS_TOKEN,
         expires_in_seconds=EXPIRES_IN_SECONDS,
         scopes=CLIENT_CREDENTIALS_SCOPES,
-        grant_type=auth.CLIENT_CREDENTIAL_GRANT,
+        grant_type=auth.CLIENT_CREDENTIALS_GRANT,
```
Comment (chardane) [inline tests/test_auth.py]: nice catch

Comment (chardane) [reply tests/test_auth.py]: will this break existing implementations?

Comment (dustinwhittle) [reply tests/test_auth.py]: I don't think anyone uses CLIENT_CREDENTIAL_GRANT externally as all the endpoints previously required authorization code grant. Either way I will bump to a major version with to indicate bc change.

**Pull request merged**